internal/app/services/core/payments/appointment_payment_helpers.go

```
@@ -427,6 +420,163 @@ func (uc *paymentUsecase) notifyProviderAsync(
+		if err != nil {
+			return &resourceFetchError{resource: "invoice", err: err}
+		}
+		fetchedInvoices = inv
+		return nil
+	})
+
+	g.Go(func() error {
+		sched, err := uc.ScheduleFhirClient.FindScheduleByPractitionerRoleID(gctx, practitionerRoleID)
+		if err != nil {
+			schedulesErr = err
+			return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 502) but it returns nil

(nilerr)